Please update the disclosure date to ISO 8601 format as `2019-09-04`.

```
[*] Running msftidy.rb in ./.git/hooks/post-merge mode
--- Checking new and changed module syntax with tools/dev/msftidy.rb ---
modules/auxiliary/dos/http/metasploit_httphandler_dos.rb - [ERROR] Incorrect disclosure date format
modules/auxiliary/dos/http/metasploit_httphandler_dos.rb - [WARNING] Please add a newline at the end of the file
```

--- a/modules/auxiliary/dos/http/metasploit_httphandler_dos.rb
+++ b/modules/auxiliary/dos/http/metasploit_httphandler_dos.rb
@@ -45,7 +45,7 @@ class MetasploitModule < Msf::Auxiliary
                       'References' => [
                           ['CVE', '2019-5645']
                       ],
-                      'DisclosureDate' => '04 September 2019'
+                      'DisclosureDate' => '2019-09-04'
           ))
 
     register_options(

